### ISSUE TITLE
How should the MQTT Publish look like ?

### ISSUE THREAD

**Issue by RonaldsGit** [Marv2190/venus.dbus-MqttToGridMeter#6]
Hello,

i get successfull the MQTT running in my Venus OS , i see the MQTT Energymeter . But all Values are "0" .
I Think i have to format the Publish String correctly .

![image](https://user-images.githubusercontent.com/130551221/231411805-df039432-4493-4607-a441-9272582c7279.png)

I tried some things but it doesn´t work.

Here some examples shown in the Mqtt Explorer :

sensor = value" "Power_curr" -514.0
= value"1234","Power_curr" -428.0
Zaehler = value"1234","Power_curr" -507.0

i want to transfer only the actual Power , to get the ESS running on the Venus os.

best regards Ronald

**Comment by J4nsen** [comment]
I dont think that the author is still active here. but you could check out this mqtt meter: https://github.com/mr-manuel/venus-os_dbus-mqtt-grid

**Comment by Marv2190** [comment]
You should get a json on:
zaehler/h_1evu/SENSOR
with the following included 
["Zaehler"]["Power_curr"])
["Zaehler"]["Total_in"])
["Zaehler"]["Total_out"])

These can then used.

But for easier use, try the DummyMeter and fill that with NodeRed (LargeImage). Much easier and better to understand.
https://github.com/Marv2190/Victron_DummyMeter_NodeRed
